Src/library/base/elks/support/reflected_copy_semantics_object.e

```diff
@@ -103,29 +105,17 @@ feature {REFLECTED_OBJECT} -- Access
 		note
 			compiler: no_gc
 		do
-			Result := dereference (referring_object.object_address, referring_physical_offset) + physical_offset
+			Result := {ISE_RUNTIME}.raw_reference_field_at (physical_offset, referring_object.object_address, referring_physical_offset)
```

Comment:
It's pretty similar to what we have in {REFLECTED_REFERENCE_OBJECT}.object_address.